pkg/kgo/client.go

```diff
@@ -129,22 +130,30 @@ func NewClient(opts ...Opt) (*Client, error) {
 	}
 	seeds := make([]hostport, 0, len(cfg.seedBrokers))
 	for _, seedBroker := range cfg.seedBrokers {
-		addr := seedBroker
-		port := int32(9092) // default kafka port
-		if colon := strings.IndexByte(addr, ':'); colon > 0 {
-			port64, err := strconv.ParseInt(addr[colon+1:], 10, 64)
-			if err != nil {
-				return nil, fmt.Errorf("unable to parse addr:port in %q", seedBroker)
-			}
-			addr = addr[:colon]
-			port = int32(port64)
+		shostport := seedBroker
+		sport := "9092" // default kafka port
+		colon := strings.LastIndexByte(shostport, ':')
+		if colon == -1 { // no port in address, append default port
+			shostport += fmt.Sprintf(":%s", sport)
```

Comment:
I think we can get rid of `sport` and have this be just `shostport += ":9092"` here and below, since sport is static and only used twice.